### ISSUE TITLE
AutoMapPins incompatible

### ISSUE THREAD

**Issue by Cronosus** [WeylandMod/WeylandMod#13]
Is possible make it compatible with AutoMapPins?
https://www.nexusmods.com/valheim/mods/195

it spam this error in console and its unable to place visible pins (they shown only after restart)
not sure whre is problem, so i reporting it on both sides. i think AutoMapPins pins shouldnt be shared, because theyre only temporary.
```
[Error  : Unity Log] ArgumentException: The Object you want to instantiate is null.
Stack trace:
UnityEngine.Object.Instantiate[T] (T original) (at <51725b129ecd4967a9a213c1ae65b295>:0)
(wrapper dynamic-method) Minimap.DMD<Minimap::UpdatePins>(Minimap)
(wrapper dynamic-method) Minimap.DMD<Minimap::Update>(Minimap)
(wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition.Trampoline<Minimap::Update>?1925344256(Minimap)
WeylandMod.Features.SharedMap.MinimapComponent.UpdateHook (On.Minimap+orig_Update orig, Minimap self) (at <46819b08cb3f40dfa3c84d0e0d72babc>:0)
(wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition.Hook<Minimap::Update>?-734199936(Minimap)
```

**Comment by glcoder** [comment]
I'll check this. Meanwhile you can check just realeased version 0.3.0 with Item Radar, maybe this can replcae AutoMapPins for you.

**Comment by glcoder** [comment]
I guess this problem was fixed in the version 0.3.0. Can you check and confirm?

**Comment by Cronosus** [comment]
yes, its ok now
thanks. but now that you have that function too, then i dont need that mod anymore anyway xD
